01_prepare_host.sh

```diff
@@ -38,20 +38,36 @@ if ! which kustomize 2>/dev/null ; then
     sudo mv kustomize /usr/local/bin/.
 fi
 
-# Download Ironic binary and Ironic inspector image
+# Clean-up any old ironic containers
+for name in ironic ironic-inspector dnsmasq httpd mariadb ipa-downloader; do
+    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill $name
```

Comment:
maybe use pgrep instead of ps | grep ?
Also I advise against using name for killing, I would use pid

Reply:
This is docker ps (or podman ps) not ps itself